### ISSUE TITLE
Look for dlls in roaming folders

### ISSUE THREAD

**Issue by JB-AM** [xlwings/xlwings#1803]
#### Describe your issue 
In Excel xlwings ribbon, click on `Import functions`, get error message : `Could not load …dlls from either the following folders: C:\program files\python39; C:/program files`
I am not an administrator of my machine, could still install xlwings, but the dlls are located there: `C:\Users\redacted\AppData\Roaming\Python` and I cannot move them to `C:/program files`, that would need the help of an administrator, something that will raise questions. 

Would it be possible to look for the dll in the user \Roaming folder if they are not found in the Program files one?

**Comment by fzumstein** [comment]
Yeah, to be honest, I think the dll search path handling should be refactored altogether. But what you can do to make this work today is to go into the xlwings add-in (password is `xlwings`), then change the following line:

```vb.net
If LoadLibrary(ParentFolder(PYTHON_WIN) + "\" + XLPyDLLName) = 0 Then
```

into

```vb.net
If LoadLibrary("C:\Users\redacted\AppData\Roaming\Python" + "\" + XLPyDLLName) = 0 Then
```

See [here](https://app.xltrail.com/#/workbook/github.com%2FZoomerAnalytics%2Fxlwings.git%2Fxlwings%2Faddin%2Fxlwings.xlam/module/Main/?branch=main&blob=af0ffe49d706f0a21a4a477a783bc691b9b08f57&version=92a6ab501d8f5cdb0ef047014c7710f66cd02bc4&path=xlwings%2Faddin%2Fxlwings.xlam%2FMain.bas&public=true&scroll=%23module-Main%20.line-num1.linenum-458) for the context.

**Comment by fzumstein** [comment]
See also https://github.com/xlwings/xlwings/issues/1538
